test/runtests.jl

```diff
@@ -8,6 +8,15 @@ using SparseArrays
 using DelimitedFiles
 using CSV
 
+@testset "Humann2" begin
+    df = CSV.read("files/humanntestfile.tsv", DataFrame, delim='\t', header=["#GeneFamily", "SRS014459-Stool_Abundance"], datarow = 2)
```

Comment:
This should probably have a function like `metaphlan_table<s>` but for `humann` instead of this at some point, but deal with that later #23